index.html

```diff
@@ -786,10 +786,10 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 initially 0.</li>
 <li>A <dfn>resource timing buffer full flag</dfn> which is
 initially false.</li>
-<li>A <dfn>resource timing secondary buffer current size</dfn> which is
-initially 0.</li>
-<li>A <dfn>secondary entry buffer</dfn> to store <a>PerformanceEntry</a>
-objects that is initially empty.</li>
+<li>A <dfn>resource timing secondary buffer current size</dfn>
+which is initially 0.</li>
+<li>A <dfn>resource timing secondary buffer</dfn> to store
+<a>PerformanceEntry</a> objects that is initially empty.</li>
```

Comment:
nit: it stores PerformanceResourceTiming objects in particular.

Reply:
true